demos/https/iot_demo_https_s3_download_async.c

```diff
@@ -648,10 +1127,11 @@ int RunHttpsAsyncDownloadDemo( bool awsIotMqttMode,
         IOT_SET_AND_GOTO_CLEANUP( EXIT_FAILURE );
     }
 
-    /* Connect to S3. */
+    /* Connect to S3. To lower demo failures due to transient network connection errors, the connection is re-attempted
+     * at a fixed interval a few times. */
     for( connAttempt = 1; connAttempt <= IOT_DEMO_HTTPS_CONNECTION_NUM_RETRY; connAttempt++ )
     {
-        httpsClientStatus = IotHttpsClient_Connect( &connHandle, &connConfig );
+        httpsClientStatus = IotHttpsClient_Connect( &_connHandle, &_connConfig );
```

Comment:
The continue statement below is redundant?

Reply:
Yes it is thanks.